dialogue-apache-hc5-client/src/main/java/com/palantir/dialogue/hc5/InstrumentedManagedHttpConnectionFactory.java

```diff
@@ -28,18 +29,34 @@ final class InstrumentedManagedHttpConnectionFactory implements HttpConnectionFa
 
     private final HttpConnectionFactory<ManagedHttpClientConnection> delegate;
     private final Timer serverTimingOverhead;
+    private final Timer socketBindSuccessesTimer;
+    private final Timer socketBindFailureTimer;
 
     InstrumentedManagedHttpConnectionFactory(
             HttpConnectionFactory<ManagedHttpClientConnection> delegate,
             TaggedMetricRegistry metrics,
             String clientName) {
         this.delegate = delegate;
         this.serverTimingOverhead = DialogueClientMetrics.of(metrics).serverTimingOverhead(clientName);
+        this.socketBindSuccessesTimer = DialogueClientMetrics.of(metrics)
+                .connectionSocketBind()
+                .clientName(clientName)
+                .result(ConnectionSocketBind_Result.SUCCESS)
+                .build();
+        this.socketBindFailureTimer = DialogueClientMetrics.of(metrics)
```

Comment:
Nit: let's factor out the three `DialogueClientMetrics.of(metrics)` into a local variable which we can reuse for each of the three metric instances we register.

Reply:
done in ca0dd326